tests/values.go

```diff
@@ -385,6 +392,15 @@ func (c *Chart) Render(values *CoderValues, options *chartutil.ReleaseOptions, c
 	return objs, nil
 }
 
+func RenderChart(t testing.TB, values *CoderValues) []runtime.Object {
```

Comment:
could also make this a func on the Chart object like MustRender or something, I guess

Reply:
good idea